plugins/TodayAtMun/diary_parser.py

```diff
@@ -0,0 +1,39 @@
+from bs4 import BeautifulSoup
+import requests
+import os
+from os import path
+import json
+from pathlib import Path
+
+
+class diary_parser:
```

Comment:
```suggestion
class DiaryParser:
```
Class names are PascalCase